docker/docker-compose.yml

```diff
@@ -2,139 +2,104 @@ version: '2'
     depends_on:
-      - stb-postgres
-      - stb-rabbitmq
+      - postgres
+      - rabbitmq
     networks:
       - stb-backend
       - stb-frontend
     volumes:
+      - ./../:$SRC_DIR
       # contains refarm-site modules
       - $DEPS_DIR
-    volumes_from:
-      - stb-source
-
+      # Thus, you can work with apps related to the refarm-site
+      # - $REFARM_SRC/stbarch:/root/.local/lib/python3.6/site-packages/stbarch
     # @todo #150 Validate if `.env` is full
     #  Is full means '.env` contains all keynames from `.env.dist`
     command: python manage.py runserver 0.0.0.0:$VIRTUAL_HOST_PORT
 
-  stb-nodejs:
-    build:
-      context: ..
-      dockerfile: docker/node/Dockerfile
-    image: fidals/stb-nodejs:latest
-    container_name: stb-nodejs
-    # (gulp build && gulp watch) - it's convenient to get built static js before start watching
-    command: bash -c '(gulp build && gulp watch) || (npm install && npm install -g gulp-cli && gulp build)'
+  # @todo #142:60m Resurrect `npm install` and `gulp build`
+  nodejs:
+    image: fidals/stb-nodejs:dev
     volumes_from:
-      - stb-python
-      - stb-source
-    environment:
-      - DEPS_DIR=$DEPS_DIR
+      - app
+    env_file:
+      - env_files/paths
 
-  stb-postgres:
+  postgres:
     image: postgres:9.5
     restart: always
-    container_name: stb-postgres
-    environment:
-      POSTGRES_DB: $DB_NAME
-      POSTGRES_PASSWORD: $DB_PASS
+    env_file:
+      - env_files/credentials
     networks:
       - stb-backend
     volumes:
-      - /opt/database/stroyprombeton/:/var/lib/postgresql/data
+      - $POSTGRES_DATA_DIR:/var/lib/postgresql/data
 
-  stb-rabbitmq:
+  rabbitmq:
     image: rabbitmq:management-alpine
     restart: always
-    container_name: stb-rabbitmq
-    environment:
-      RABBITMQ_DEFAULT_USER: $RABBITMQ_DEFAULT_USER
-      RABBITMQ_DEFAULT_PASS: $RABBITMQ_DEFAULT_PASS
+    env_file:
+      - env_files/credentials
     networks:
       - stb-backend
+    # @todo #142:30m Check why rabbitmq exposes it's ports to host machine.
+    #  And write doc or request some fix.
     ports:
-      - '5674:5672'
-      - '15674:15672'
-
-  # TODO:
-  # Selenium Grid stuff (can't get it to work)
-
-  # selenium-hub:
-  #   image: selenium/hub:3.2.0
-  #   restart: always
-  #   networks:
-  #     - stb-backend
-  #   ports:
-  #     - '4444:4444'
+      - 5674:5672
+      - 15674:15672
 
-  # selenium-chrome:
-  #   image: selenium/node-chrome:3.2.0
-  #   restart: always
-  #   depends_on:
-  #     - selenium-hub
-  #   environment:
-  #     - HUB_PORT_4444_TCP_ADDR=selenium-hub
-  #     - HUB_PORT_4444_TCP_PORT=4444
-  #   networks:
-  #     - stb-backend
-  #   ports:
-  #     - 5555
-  #     - 5900
-
-  stb-selenium:
-    image: selenium/standalone-chrome-debug:3.6.0
+  selenium:
+    image: selenium/standalone-chrome-debug:3.10.0
     restart: always
     ports:
-      - '4444'
-      - '5900:5900'
+      - 4444:4444
+      - 4444
```

Comment:
`- 4444:4444` will be enough

Reply:
@artemiy312 , i'll leave just `- 4444` for safety

TODO